src/sentry/grouping/ingest/config.py

```diff
@@ -17,24 +17,18 @@
 
 Job = MutableMapping[str, Any]
 
-# We are moving all projects off these configuration without waiting for events
-CONFIGS_TO_DEPRECATE: list[str] = [
-    "mobile:2021-02-12",
-]
+# Used to migrate projects that have no activity via getsentry scripts
+CONFIGS_TO_DEPRECATE = "mobile:2021-02-12"
```

Comment:
Why switch this not to being a list (tuple, set, whatever)? There might be multiple in the future, no?